oxfmtrc.jsonc

```diff
@@ -8,7 +8,7 @@
     "**/generated/**",
     // Ignore `fixtures` directories except for `apps/oxlint/test/fixtures`
     "**/fixtures/**",
-    "!apps/oxlint/test/fixtures/**",
+    "!apps/oxlint/test/fixtures/**/plugin.ts",
```

Comment:
This doesn't work. Linted files are ignored as intended, but `plugin.ts` are not negated, they are also ignored. Not sure why. Single asterisk has the same issue.